kf_model_fhir/ingest_plugin/target_api_builders/sequencing_task.py

```diff
@@ -0,0 +1,172 @@
+"""
+Builds FHIR Task resources (https://www.hl7.org/fhir/Task.html)
```

Comment:
Maybe you already have this in another place in the model, but if not, can you describe what the sequencing task represents/what it is?